2.7/s2i/bin/run

```diff
@@ -15,6 +15,15 @@ function should_migrate() {
 
 set -e
 
+APP_FILE="${APP_FILE:-app.py}"
+if [[ "$APP_FILE" ]]; then
```

Comment:
This will never be false, since you're defaulting it one line before, credits to @bparees.
The same applies to all other python versions.